test/acceptance/features/addRBACForOperator.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Making Service Biniding Operator RBAC pluggable so that other controllers/admins can add additional rules for the service binding operator.
+    @rbactest
```

Comment:
do we need an extra annotation here?

Reply:
no this was for running the tests locally. reverting